src/main/kotlin/com/bantolomeus/controller/BookController.kt

```diff
@@ -15,7 +15,7 @@ class BookController(private val bookService: BookService) {
         return bookService.createBook(book)
     }
 
-    @PutMapping(consumes = ["application/json;charset=UTF-8"], path = ["{bookName}"])
```

Comment:
`content-type:application/json` is required by default or so, right? Spring or Tomcat or whoever checks for the header wouldn't let me put anything here even without the `consumes` config here

Reply:
Yes. I tried it out and I got a Json parse error.

Reply:
Correct